recipes/meme/meta.yaml

```diff
@@ -21,23 +23,27 @@ requirements:
         - python
         - perl
         - ghostscript
-        - zlib
+        - zlib {{ CONDA_ZLIB }}*
         - libxslt
         - libxml2
         - expat
         - perl-xml-parser
         - perl-app-cpanminus
+        - perl-module-build
         - perl-yaml
         - perl-xml-simple
         - perl-html-template
         - perl-cgi
+        - perl-html-parser
+        - perl-html-tree
+
     run:
         - libgcc # [not osx]
         - yaml
         - expat
         - python
         - perl
-        - zlib
+        - zlib {{ CONDA_ZLIB }}*
         - libxslt
         - libxml2
```

Comment:
we should probably pin this as well to: libxml2 2.9.*

Reply:
Thanks @bgruening,  pinning it in the new commit.